readme.md

```diff
@@ -393,9 +393,11 @@
 - [Honeypots](https://github.com/paralax/awesome-honeypots) - Deception trap, designed to entice an attacker into attempting to compromise the information systems in an organization.
 - [Incident Response](https://github.com/meirwah/awesome-incident-response)
 - [Vehicle Security and Car Hacking](https://github.com/jaredmichaelsmith/awesome-vehicle-security)
+- [Security Write-Ups and POCs](https://github.com/dhaval17/awsome-security-write-ups-and-POCs) - Curated list of write-ups and POCs from the world of information security.
```

Comment:
The description here is just repeating the title. Please read my comment again in https://github.com/sindresorhus/awesome/pull/882#discussion_r112399953 It should be about the theme, not your list.